packages/webapi/models/L1.go

```diff
@@ -32,6 +32,7 @@ type StateAnchor struct {
 	Raw           string `json:"raw" swagger:"desc(The raw data of the anchor (Hex)),required"`
 }
 
+// TODO should be removed in the future
```

Comment:
Why should it be removed?

Reply:
The user who want to examine the past versions of Anchor can use `iota_tryGetPastObject`. And we already have a metric about `StateAnchor`. Not sure how much useful that a metric of StateTransaction is